src/components/SelectComponents/ComboboxMulti/ComboboxMulti.tsx

```diff
@@ -76,6 +76,7 @@ export function ComboboxMulti<S>({
                 <IoMdArrowDropup
                     {...elem('dropdownIcon', { disabled })}
                     {...getToggleButtonProps({ disabled, onClick: blur })}
+                    role="img"
```

Comment:
I think this should have `role="button"`

Reply:
:+1: